environment.py

```diff
@@ -605,6 +605,13 @@ def env() -> Mapping[str, Optional[str]]:
         #
         'TF_DATA_DIR': '{project_root}/deployments/.active/.terraform.{AWS_PROFILE}',
 
+        # Make Terraform's deprecation warnings more compact
+        #
+        **{
+            'TF_CLI_ARGS_' + stage: '-compact-warnings'
+            for stage in ['validate', 'plan', 'apply']
```

Comment:
```suggestion
            'TF_CLI_ARGS_' + command: '-compact-warnings'
            for command in ['validate', 'plan', 'apply']
```

"stage" is already taken and `terraform --help` refers to them as "commands".